build/init-topics-and-clients.sh

```diff
@@ -40,6 +40,11 @@ do
     base=${fname##*/}
     topic=${base%.*}
 
+    # Skip faq.md so it is never overwritten
```

Comment:
I don't think this is the correct approach - this basically bypasses the usefulness of the SSG.